src/renderer/entities/transaction/ui/QrCode/QrReader/QrReader.tsx

```diff
@@ -337,7 +331,7 @@ export const QrReader = ({
     })();
 
     return () => {
-      stopScanning();
+      handleStopScanning();
```

Comment:
We can simply call `stopScanning({ streamRef, controlsRef, bgControlsRef });`
No need for a separate function
Same for other places